package.json

```diff
@@ -37,5 +37,8 @@
   },
   "peerDependencies": {
     "cql-execution": ">=1.3.0 || ^3.0.0-beta"
+  },
+  "resolutions": {
```

Comment:
I'm assuming resolutions is used because of `yarn` not providing a convenient way to resolve vulnerabilities in `yarn.lock` like you can with `npm` a la `npm audit fix`?

To fix vulns in this repo in the past, I think I've used the [yarn-audit-fix package](https://www.npmjs.com/package/yarn-audit-fix) (crazy to me that something like this even _needs_ to exist). Not great, but it does avoid the need for `resolutions` in this case by just modifying `yarn.lock` directly like `npm audit fix` would.

Curious to @cmoesel's thoughts

Reply:
I've used resolutions in the past this way; basically like a manual `npm audit fix`.  Open to other approaches.  A nice thing about resolutions is that you have a pretty clear paper trail of fixes you've had to incorporate.